source/jormungandr/jormungandr/parking_space_availability/car/star.py

```diff
@@ -0,0 +1,118 @@
+# encoding: utf-8
+# Copyright (c) 2001-2014, Canal TP and/or its affiliates. All rights reserved.
+#
+# This file is part of Navitia,
+#     the software to build cool stuff with public transport.
+#
+# Hope you'll enjoy and contribute to this project,
+#     powered by Canal TP (www.canaltp.fr).
+# Help us simplify mobility and open public transport:
+#     a non ending quest to the responsive locomotion way of traveling!
+#
+# LICENCE: This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# IRC #navitia on freenode
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+from __future__ import absolute_import, print_function, unicode_literals, division
+
+import logging
+import pybreaker
+import requests as requests
+import jmespath
+
+from jormungandr import cache, app, utils, new_relic
+from jormungandr.parking_space_availability import AbstractParkingPlacesProvider
+from jormungandr.parking_space_availability.car.parking_places import ParkingPlaces
+from jormungandr.ptref import FeedPublisher
+
+DEFAULT_STAR_FEED_PUBLISHER = None
+
+
+class StarProvider(AbstractParkingPlacesProvider):
+
+    WS_URL_TEMPLATE = 'http://data.explore.star.fr/api/records/1.0/search/?dataset={}&refine.idparc={}'
```

Comment:
this will be done only for STAR in this PR, another PR will be opened for jcdecaux